src/Parser/EnhancementShaman/CombatLogParser.js

```diff
@@ -0,0 +1,336 @@
+    const furyofairUptime = this.selectedCombatant.getBuffUptime(SPELLS.FURY_OF_AIR.id) / this.fightDuration;
+
+    const nonDpsTimePercentage = this.modules.alwaysBeCasting.totalDamagingTimeWasted / fightDuration;
+    const deadTimePercentage = this.modules.alwaysBeCasting.totalTimeWasted / fightDuration;
+
+    const masteryPercent = this.selectedCombatant.masteryPercentage;
+    const hastePercent = this.selectedCombatant.hastePercentage;
+
+    //Legendaries
+    // const hasUncertainReminder = this.selectedCombatant.hasHead(ITEMS.UNCERTAIN_REMINDER.id);
+    // const hasEmalons = this.selectedCombatant.hasChest(ITEMS.EMALONS_CHARGED_CORE.id);
+    const hasAkainus = this.selectedCombatant.hasWrists(ITEMS.AKAINUS_ABSOLUTE_JUSTICE.id);
+    // const hasSmolderingHeart = this.selectedCombatant.hasHands(ITEMS.SMOLDERING_HEART.id);
+    // const hasStorm = this.selectedCombatant.hasWaist(ITEMS.STORM_TEMPESTS.id);
+    // const hasRoots = this.selectedCombatant.hasLegs(ITEMS.ROOTS_OF_SHALADRASSIL.id);
+    // const hasSpiritual = this.selectedCombatant.hasFeet(ITEMS.SPIRITUAL_JOURNEY.id);
+    // const hasSoul = this.selectedCombatant.hasFinger(ITEMS.SOUL_OF_THE_FARSEER.id);
+    const hasEOTN = this.selectedCombatant.hasFinger(ITEMS.EYE_OF_THE_TWISTING_NETHER.id);
+    // const hasSephuz = this.selectedCombatant.hasFinger(ITEMS.SEPHUZS_SECRET.id);
+
+  this.selectedCombatant._combatantInfo.gear.forEach(function(value) {
+    const equippedItem = getItemInfo(value.id);
+
+    if(equippedItem !== undefined && equippedItem.quality === 5) {
+      results.items.push({
+        item: equippedItem,
+        result: (
+          <dfn data-tip="">
+            Equipped Legendary
+          </dfn>
+        ),
+      });
+    }
+  });
+
+  //add trinkets to item list
+  function getItemInfo(id) {
+    let value;
+    Object.keys(ITEMS).some(function(k) {
+      if(ITEMS[k].id === id) {
+        value = ITEMS[k];
+      }
+    });
+
+    return value;
+  }
+
+    if (nonDpsTimePercentage > 0.3) {
+      results.addIssue({
+        issue: `[NYI] Your non DPS time can be improved. Try to cast damaging spells more regularly (${Math.round(nonDpsTimePercentage * 100)}% non DPS time).`,
+        icon: 'petbattle_health-down',
+        importance: getIssueImportance(nonDpsTimePercentage, 0.4, 0.45, true),
+      });
+    }
+    if (deadTimePercentage > 0.2) {
+      results.addIssue({
+        issue: `Your dead GCD time can be improved. (${Math.round(deadTimePercentage * 100)}% dead GCD time).`,
+        icon: 'spell_mage_altertime',
+        importance: getIssueImportance(deadTimePercentage, 0.35, 0.4, true),
+      });
+    }
+
+    if(flametongueUptime < .85) {
+		results.addIssue({
+        issue: `Your Flametongue uptime of ${formatPercentage(flametongueUptime)}% is below 85%`,
+        icon: SPELLS.FLAMETONGUE_BUFF.icon,
+        importance: getIssueImportance(flametongueUptime, 0.35, 0.4, true),
```

Comment:
You seem to have forgotten to change these values; they determine the suggestion importance. An example value could be `getIssueImportance(ruleOfLawUptime, 0.8, 0.7)`.